src/services/anchor/index.ts

```diff
@@ -61,19 +64,71 @@ export const fetchTomlValues = async (TOML_FILE_URL: string): Promise<TomlValues
   };
 };
 
-async function getUrlParams(ephemeralAccount: string, supportsClientDomain: boolean): Promise<URLSearchParams> {
-  if (supportsClientDomain) {
-    return new URLSearchParams({ account: ephemeralAccount, client_domain: config.applicationClientDomain });
+//A memo derivation. TODO: Secure? how to check for collisions?
```

Comment:
Hash the address, and get the numeric representation of the hash. Apparently the memo needs to be numeric.